multigeiger/speaker.cpp

```diff
@@ -45,7 +45,7 @@ void IRAM_ATTR isr_audio() {
 
   // tone and tick generation, also led blinking
   int frequency_mHz = 0, volume = 0, led = 0, duration_ms = 0;  // init avoids compiler warning
-  static bool playing_audio = false, playing_tick = false;
+  bool playing_audio = false, playing_tick = false;
```

Comment:
i don't think this would work. it will be set to true when playing the first tone and at the end of that interrupt service, the local variable will be lost. when it plays the next tone, it will start as false again.
line 52..60 won't be executed because isr_sequence is not NULL for the non-first tone.

Reply:
this might explain why it repeats all the time any sequence

Reply:
ok, should be fixed, code runs here also with ticks / start_sound enabled now 😙🎶